apps/src/templates/HeaderBannerNoImage.jsx

```diff
@@ -0,0 +1,53 @@
+// Use this component if a dashboard page has a header banner with no image.
+// Adapted from the HeaderBanner component.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import Typography from '@cdo/apps/componentLibrary/typography';
+import style from './header-banner-no-image.module.scss';
+
+export default class HeaderBanner extends React.Component {
+  static propTypes = {
+    headingText: PropTypes.string,
+    subHeadingText: PropTypes.string,
+    description: PropTypes.string,
+    backgroundColor: PropTypes.object,
+    children: PropTypes.node,
+  };
+
+  render() {
+    const {
+      headingText,
+      subHeadingText,
+      description,
+      backgroundColor,
+      children,
+    } = this.props;
+
+    const backgroundStyling = {
+      backgroundColor: backgroundColor,
+      ...backgroundStyling,
+    };
+
+    return (
+      <section style={backgroundStyling} className={style.banner}>
+        <div className={style.wrapper}>
+          <Typography semanticTag="h1" visualAppearance="heading-xxl">
```

Comment:
nit: You can use the shorthand `Heading1` here (see https://github.com/code-dot-org/code-dot-org/blob/0dd46f09187dbe0656cc93ad45025e8423c980f3/apps/src/componentLibrary/typography/TypographyElements.tsx#L12).

Below, you can use `BodyOneText` as the shorthand.